google-map.html

```diff
@@ -173,6 +228,21 @@
         });
         this.contentChanged();
         setupDragHandler_.bind(this)();
+
+        this._forwardEvent('click');
+        this._forwardEvent('dblclick');
+        this._forwardEvent('mousedown');
+        this._forwardEvent('mousemove');
+        this._forwardEvent('mouseout');
+        this._forwardEvent('mouseover');
+        this._forwardEvent('mouseup');
+        this._forwardEvent('rightclick');
+      },
+
+      _forwardEvent: function(name) {
+        google.maps.event.addListener(this.marker, name, function(event) {
+          this.fire('google-map-marker-' + name, event);
```

Comment:
I chose the more verbose form to allow a developer to register handlers above the element (e.g. on the google-map itself) and still disambiguate the source by type.  However, there is also value in brevity (and the handler will be passed the source element).  Do you have a recommendation @ebidel?